README.md

```diff
@@ -1,3 +1,131 @@
 # testcafe-browser-natives
 
 [![Build Status](https://travis-ci.org/superroma/testcafe-browser-natives.svg)](https://travis-ci.org/superroma/testcafe-browser-natives)
+
+Testcafe Browser Natives is a TestCafe library for performing platform-dependent actions on browser windows.
+#Build Process
+To build the binaries from source files, execute the gulp task corresponding to your operating system:
+```
+'copy-win-executables'
+'build-mac-executables'
+'copy-mac-script'
+```
+Note that the application for a particular platform must be built on a machine with the same platform.
+You can find the pre-build native binaries in the bin directory. So, you do not need to build them yourself.
```

Comment:
We need some kind of design note why we publish prebuilt binaries for all platform, since it's not a common practice. 

`So, you do not need to build them yourself.`

I'd rather remove this sentence.